app/lib/bk/compat/parsers/bitrise/workflows.rb

```diff
@@ -14,20 +16,43 @@ def parse_workflow(wf_name, wf_config)
           step_key, step_config = step.first
           cmd_step << translate_step(step_key, step_config)
         end
-
+        process_workflow_envs(wf_config, cmd_step)
         cmd_step
       end
 
+      def process_workflow_envs(wf_config, cmd_step)
+        cmd_step.env ||= {}
```

Comment:
commands class already take care of this, there is no need to ensure it is a hash that is empty